### ISSUE TITLE
feat: Add new value for a default policy module repository

### ISSUE THREAD

**Issue by viccuad** [kubewarden/helm-charts#242]
Sometimes it is useful to provide a different default registry for the policies, separate from normal images.
This for example happens if the user wants an http registry for the wasm modules because their image repository doesn't support OCI artifacts yet.

### Acceptance criteria

- [x] On kubewarden-defaults chart (the only one container policies so far), add a new value field for setting the default *module policy* registry. This is, in addition to `.global.cattle.systemDefaultRegistry`, and only for policies.
- [x] Consider if to make it part of `.global.cattle.*`, for example, `.global.cattle.oci.systemDefaultRegistry`, and if it needs to be shared by all chart values  (that means, set in `values.yaml` for all charts, not only in `chart-values.yaml` for kubewarden-defaults chart).
- [x] Check docs.kubewarden.io.
- [x] Check Airgap story and scripts.
- [ ] Check usage in Kubewarden UI.
- [ ] Check Kubewarden UI Airgap story.

**Comment by jvanz** [comment]
I've built a new version of the UI pointing to a new Helm chart version containing the fix from this issue. Then, I've forked the ui-plugins repo and add this new UI version on it. Therefore, I was able to test the installation on the UI. I've followed these steps:

1. Uninstall Kubewarden Helm charts from previous test
2. Install `kubewarden-controller` chart 


![Image](https://github.com/kubewarden/helm-charts/assets/1514798/bc19de57-e4a0-4957-86ad-4866be886118)


3. Install `kubewarden-defaults` chart with default container registry. But changed the default policies registry configuration



![Image](https://github.com/kubewarden/helm-charts/assets/1514798/750adc40-a4ed-4fa6-ae26-e29b25cd3f25)


4. The installation went fine,  all the resources get to active state. And the policies have been downloaded from my  "private" policies OCI artifacts


![Image](https://github.com/kubewarden/helm-charts/assets/1514798/6a1f4dec-bd6c-4e4a-8abd-a200c77381d9)
![Image](https://github.com/kubewarden/helm-charts/assets/1514798/22298f63-c093-40b1-85e2-349db4160866)

 
Therefore, I've consider this issue done. But keep in mind that there is an issue driven by @jordojordo  to further validate air gap installation on UI
